### PR TITLE
Bug 1235300 - [GijTV] Smart system has to set the ready-state attribute to let JSMarionette know ready to test

### PR DESCRIPTION
**Pull request by Fischer-L**

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=a82a8da5a81ede0eff2efeb4534fe39a719a6e94

File: tv_apps/smart-system/js/bootstrap.js [hunk 1]
```diff
@@ -38,6 +38,11 @@ window.addEventListener('load', function startup() {
   }
 
   function safelyLaunchFTU() {
+    window.addEventListener('homescreenloaded', function onHomescreenLoaded() {
+      window.removeEventListener('homescreenloaded', onHomescreenLoaded);
+      // To let Gijtv test know we are ready to test.
```
Comment (luke-chang) [inline tv_apps/smart-system/js/bootstrap.js]: nit: To let GijTV tests know we are ready to test.

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=b1e472387992d4b67981350248c77891d5081c48

File: tv_apps/smart-system/js/bootstrap.js (hunk 1 above)
Comment (Fischer-L) [inline tv_apps/smart-system/js/bootstrap.js]: Update the comment, thanks

**Pull request merged**